### DISCUSSION TITLE
Domain verification was failed

### DISCUSSION THREAD

**Discussion by mkkang01** [vercel/community#4086] [Help]
### Summary

I want to use app.hoy.im domain in my project. so i registered _vercel TXT record to my registrar. 

But it does not work for over 24 hours. 

![스크린샷 2023-09-15 오후 12 07 40](https://github.com/vercel/community/assets/83683326/618a92aa-c4c5-4491-9367-77361099b6e4)

I already check https://dnschecker.org/#TXT/_vercel.hoy.im that record is propagated.

### Steps to Reproduce

1. Register a domain for a project in your first account.
2. Register the same domain in your second account.
3. Delete the project in the first account.
4. Authenticate _vercel.hoy.im to the second account.

**Comment by amyegan** [comment]
Hi @mkkang01. If you're no longer using it on the first account, you can transfer it to the second account. No TXT record needed 🙂 

https://vercel.com/docs/projects/domains/transfer-your-domain

**Reply by Paul171** [reply]
I have the same problems, may I know how I should do if my vercel account is still using but added the TXT record doesn't work? thanks

**Reply by amyegan** [reply]
If you let me know the domain, I can look into this for you @Paul171

**Reply by Paul171** [reply]
Is it possible for me to send a message to you about this? 🙏🏻

**Reply by amyegan** [reply]
@Paul171 You can send me a message on X or Telegram `@AmyAEgan`
